man/man1/dsnode.1

```diff
@@ -0,0 +1,27 @@
+.TH dsnode 1
+
+.SH NAME
+
+dsnode - The DataStorm Node server.
```

Comment:
Should we update `dsnode` to use Ice::Service class?

Reply:
Yes, probably.